examples/with-apollo-and-redux/package.json

```diff
@@ -9,11 +9,11 @@
   "dependencies": {
     "graphql": "^0.9.3",
     "isomorphic-fetch": "^2.2.1",
-    "next": "latest",
+    "next": "^4",
```

Comment:
Then lets keep this `latest`

Reply:
sure. but when we get next 5 that might requires react 17, then we would need to do this again. But I think it's OK so users of this example would have to always use the latest React to play here